openedx/core/djangoapps/course_groups/views.py

```diff
@@ -213,6 +117,8 @@ def course_cohort_settings_handler(request, course_key_string):
         is_cohorted = request.json.get('is_cohorted')
         try:
             cohorts.set_course_cohorted(course_key, is_cohorted)
+            # TODO: this is the logic that we will want to change (only change division scheme
```

Comment:
I will change this logic and add unit test coverage in my next round of changes.